webapp/Connector/src/model/StudyAxisData.js

```diff
@@ -234,9 +240,9 @@ Ext.define('Connector.model.StudyAxisData', {
 
                 visit = study.visits[alignedDay];
 
-                if (visitTagCaption)
+                if (visitTagLabel)
                 {
-                    visit.visitTags.push(this.getVisitTag(study.name, groupLabel, visitTagCaption, groupDesc));
+                    visit.visitTags.push(this.getVisitTag(study.name, groupLabel, visitTagLabel, groupDesc));
```

Comment:
Really minor comment, probably update getVisitTag function's parameter to tagLabel instead of tagCaption to avoid confusion later on.